py/README.rst

```diff
@@ -13,7 +13,7 @@ Usage
 To install it execute::
 
     $ python3.5 -m venv venv
-    $ venv/bin/python -m pip install -r requirements.txt
+    $ venv/bin/python -m pip install -r py/requirements.txt
```

Comment:
not sure about this.. Depends if the readme is read from the root folder or if someone is already inside the py folder while reading it. Both variants are probably confusing?